Python/Product/TestAdapter/PythonRunSettings.cs

```diff
@@ -186,6 +188,8 @@ ref outObj
                             writer.WriteAttributeString("workingDir", config.WorkingDirectory);
                             writer.WriteAttributeString("interpreter", config.GetInterpreterPath());
                             writer.WriteAttributeString("pathEnv", config.Interpreter.PathEnvironmentVariable);
+                            writer.WriteAttributeString("unitTestRootDir", unitTestRootDir);
+                            writer.WriteAttributeString("unitTestPattern", unitTestPattern);
```

Comment:
when writing out the unittest properties to runtimesettings we dont try to validate and use defaults here.